cli/src/description_util.rs

```
@@ -349,11 +350,13 @@ pub fn combine_messages_for_editing(
             .into_string()
             .map_err(|_| user_error("Trailers should be valid utf-8"))?;
         let new_trailers = parse_trailers(&trailer_lines)?;
-        let trailers: String = new_trailers
+        let trailers = new_trailers
             .iter()
             .filter(|trailer| !old_trailers.contains(trailer))
-            .map(|trailer| format!("{}: {}\n", trailer.key, trailer.value))
-            .collect();
+            .fold(String::new(), |mut output, trailer| {
+                let _ = writeln!(output, "{}: {}", trailer.key, trailer.value);
```

nit: `.unwrap()` instead of ignoring error

btw, I think a better option is to filter out redundant `trailers`, and `write!()` into `combined`.

---

Something like this?

```rust
let new_trailers = parse_trailers(&trailer_lines)?;
let mut once = true;
for trailer in new_trailers {
    if !old_trailers.contains(&trailer) {
        if once {
            combined.push_str("\nJJ: Trailers not found in the squashed commits:\n");
            once = false;
        }
        writeln!(combined, "{}: {}", trailer.key, trailer.value).unwrap();
    }
}
```